app/controllers/opendoors/tools_controller.rb

```diff
@@ -51,7 +51,8 @@ def url_update
         flash[:error] = ts("There is already a work imported from the url %{url}.", url: @imported_from_url)
       else
         # ok let's try to update
-        @work.update_attribute(:imported_from_url, @imported_from_url)
+        @work.update_attribute(:imported_from_url, @imported_from_url) # TODO AO3-6591
+        WorkUrl.prepare(@work, @imported_from_url).save(validate: false)
```

Comment:
Disabling validation to be consistent with the `update_attribute` just above... but might be a bad idea actually.